sql/pg_cron-test.sql

```diff
@@ -0,0 +1,5 @@
+CREATE EXTENSION pg_cron VERSION '1.0';
+SELECT extversion FROM pg_extension WHERE extname='pg_cron';
```

Comment:
optional: it'd be pretty nice to do a `cron.schedule` and `cron.unschedule` here, just to make sure there's no crash.

Reply:
No problem, solved it here https://github.com/citusdata/pg_cron/pull/47/commits/0305160449e4ea2b7f4862ad88b26f055d7fbc74.